components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/constants/I18nMgtConstants.java

```diff
@@ -61,4 +64,62 @@ private ErrorMsg() {
         public static final String EMAIL_TEMPLATE_TYPE_NODE_FOUND = "10001";
         public static final String EMAIL_TEMPLATE_TYPE_ALREADY_EXISTS = "10002";
     }
+
+    /**
+     * Class which contains the error scenarios.
+     */
+    public static class ErrorScenarios {
+
+        // ETM - EMAIL TEMPLATE MANAGER
+        public static final String EMAIL_TEMPLATE_MANAGER = "ETM";
+    }
+
+    /**
+     * Enum which contains error codes and corresponding error messages.
+     */
+    public enum ErrorMessages {
+
+        ERROR_CODE_NULL_TEMPLATE_OBJECT("60001", "Notification template object cannot be null."),
```

Comment:
Notification template  is not provided

Reply:
Fixed: 008a5546baef7725f1a4c04554942b4db91f41f8